docs/source/index.rst

```diff
@@ -5,10 +5,16 @@
 
 Boto3 documentation
 ====================
-Boto is the Amazon Web Services (AWS) SDK for Python. It enables Python
-developers to create, configure, and manage AWS services, such as EC2 
-and S3. Boto provides an easy to use, object-oriented API, as well as 
-low-level access to AWS services.
+
+You use the AWS SDK for Python to create, configure, and manage AWS services, such as Amazon
```

Comment:
We should probably clarify the Boto3 usage in the first sentence, e.g. "You use the AWS SDK for Python (Boto3) [...]". In general we should probably trend towards using Boto3 instead of "the AWS SDK for Python" in these docs since they're already well scoped.